### PR TITLE
make http server timeout configurable

### PR DESCRIPTION
**Pull request by leon1995**
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have updated the [changelog](../CHANGELOG.md) accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,12 @@ All notable changes to the sdc11073 module will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [current]
```
Comment (maximilianpilz) [inline CHANGELOG.md:7]:
```suggestion
## [Unreleased]
```

**Review by maximilianpilz — CHANGES_REQUESTED** [review]

**Comment by maximilianpilz** [comment]
what about those sections in the initial comment that are empty ?

if you are not going to fill them, then why are they there ?

File: CHANGELOG.md (hunk 1 above)
Comment (deichmab-draeger) [reply CHANGELOG.md:7]: I think release state has nothing to do with the repo itself. I suggest [master], but [current] is also ok for me.

**Review by deichmab-draeger — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (leon1995) [reply CHANGELOG.md]: done

**Comment by leon1995** [comment]
> what about those sections in the initial comment that are empty ?
> 
> if you are not going to fill them, then why are they there ?

I removed this parts as they are not needed. Most of the information are found in the linked issue

**Review by deichmab-draeger — APPROVED** [review]

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -1,21 +1,4 @@
...
-<!--- Please link to the issue here: -->
-
-## Motivation and Context
-<!--- Why is this change required? What problem does it solve? -->
-
-## How Has This Been Tested?
-<!--- Please describe in detail how you tested your changes. -->
-<!--- Include details of your testing environment, and the tests you ran to -->
-<!--- see how your change affects other areas of the code, etc. -->
+<!--- Provide a general summary of your changes in the title above -->
```
Comment (maximilianpilz) [inline .github/pull_request_template.md:1]: you did not provide this

Comment (leon1995) [reply .github/pull_request_template.md:1]: done

**Review by maximilianpilz — APPROVED** [review]

**Pull request merged**